components/editor/editor.mjs

```diff
@@ -17,6 +17,7 @@ import {
 } from "./panels/package/package-content";
 import {indentationMarkers} from '@replit/codemirror-indentation-markers';
 import {CommandEditor} from "./command-k/command-editor";
+import {ftd} from "./ftd/language";
```

Comment:
Let's call it `./fastn/language` etc. FTD is no longer the name, we want to use fastn language terminology everywhere.